core-rust-bridge/src/main/java/com/radixdlt/protocol/ProtocolConfig.java

```diff
@@ -105,6 +119,43 @@ public static ProtocolConfig onlyGenesis() {
     return new ProtocolConfig(ImmutableList.of());
   }
 
+  public static ProtocolConfig launchAt(String upToProtocolVersionName) {
+    String newVersion = null;
+    List<ProtocolUpdateTrigger> protocolUpdateTriggers = new ArrayList<>();
+    for (String version : VERSION_NAMES) {
+      var lastVersion = newVersion;
+      newVersion = version;
+      if (lastVersion != null) {
```

Comment:
Minor: for readability, it would be better to just update the newVersion after the first if clause, instead of introudcing a new varible.